google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataClient.java

```diff
@@ -333,6 +334,33 @@ public void readRowsAsync(Query query, ResponseObserver<Row> observer) {
     return stub.mutateRowCallable();
   }
 
+  /**
+   * Mutates multiple rows in a batch. Each individual row is mutated atomically as in MutateRow,
+   * but the entire batch is not executed atomically.
+   *
+   * <p>Sample code:
+   *
+   * <pre>{@code
+   * InstanceName instanceName = InstanceName.of("[PROJECT]", "[INSTANCE]");
+   * try (BigtableClient bigtableClient = BigtableClient.create(instanceName)) {
+   *   try (BulkMutations mutations = bigtableClient.newBulkMutations()) {
+   *     for(String someValue in someCollection) {
```

Comment:
Use `:` instead of `in`, add a space after `for`

Reply:
done